internal/controllers/topology/cluster/reconcile_state_test.go

```diff
@@ -1459,69 +1488,6 @@ func TestReconcileReferencedObjectSequences(t *testing.T) {
 				},
 			},
 		},
```

Comment:
Why did we drop this test case? Behavior change?

Reply:
This test made sense before because we were using custom code specifically to handle labels in an authoritative way (and this code was also sensible to label.with.dots).
Now labels are managed via SSA like any other field, so no need for a specif test

Reply:
Okay let's discuss authoritative independently I have it on my list